src/Extend/Event.php

```diff
@@ -34,12 +35,32 @@ public function listen(string $event, $listener)
         return $this;
     }
 
+    /**
+     * Add a subscriber for a set of domain events dispatched by flarum or a flarum extension.
+     *
+     * @param string $subscriber
```

Comment:
Should this parameter be a bit more documented, like `listen` ?

I see Laravel's own phpdoc for `subscribe` is also lacking a proper description. But they do allow `object|string`.

Since there's no interface for listeners I'm not exactly sure what we can say, but we should at least specify it's a `::class` parameter.